File: sandbox/php/src/EmbeddedSignUrlDefaultExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();

try {
    $response = (new Dropbox\Sign\Api\EmbeddedApi(config: $config))->embeddedSignUrl(
        signature_id: "50e3542f738adfa7ddd4cbd4c00d2a8ab6e4194b",
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling Embedded#embeddedSignUrl: {$e->getMessage()}";
}

File: sandbox/ruby/src/EmbeddedSignUrlDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::EmbeddedApi.new.embedded_sign_url(
        "50e3542f738adfa7ddd4cbd4c00d2a8ab6e4194b",
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling Embedded#embedded_sign_url: #{e}"
end
